### ISSUE TITLE
(minor) Inconsistent format of screenshot-thumbnails and final-screenshot

### ISSUE THREAD

**Issue by alekseykulikov** [GoogleChrome/lighthouse#7092]
#### What is the current behavior?
It's a minor inconsistency, but `screenshot-thumbnails` and `final-screenshot` have the same type of data but in different format.

```json
{
  "screenshot-thumbnails": {
     "details": {
       "items": [
          {
            "timing": 3335,
            "timestamp": 7104511246,
            "data": "/9j/4AAQSkZJRgABAQAAAQABAAD..."
          }
        ]
     }
  },
  "final-screenshot": {
     "details": {
        "type": "screenshot",
        "timestamp": 7104882.46,
        "data": "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQ..."
      }
   }
}
```

#### What is the expected behavior?
It would be better to have the new `final-screenshot` in the same format as `screenshot-thumbnails`:
- `data` is a valid base64 encoded jpeg (remove `data:image/jpeg;base64,`)
- `timing` contains the actual value
- `timestamp` in the same scale as other timestamps (`timestamp` * 1000)

#### Environment Information
* Lighthouse version: 4.0.0

**Comment by patrickhulce** [comment]
Thanks for pointing this out @alekseykulikov! Unfortunately we already have consumers relying on the format in `final-screenshot` :/

Maybe we can put this into 5.0...

Based on the [review](https://github.com/GoogleChrome/lighthouse/pull/5673#discussion_r203497419) it looks like it started out as the same as our filmstrip but changed formats before landing. @brendankenny do you recall if there was a PSI reason that prompted the switch?

**Comment by patrickhulce** [comment]
Fixed by #8299
